src/Mappers/TransactionMapper.php

```diff
@@ -263,6 +263,20 @@ private static function getField(BaseTransaction $transaction, \DOMElement $elem
         return $fieldElement->textContent;
     }
 
+    private static function getFieldAsMoneyObject(
+        BaseTransaction $transaction,
+        \DOMElement $element,
+        string $fieldTagName
+    ): ?Money {
+        $fieldValue = self::getField($transaction, $element, $fieldTagName);
+
+        if ($fieldValue === null) {
+            return null;
+        }
+
+        return Money::EUR((string)(100 * $fieldValue));
```

Comment:
This EUR centricity is not alright. The method should take the currency as an argument.

Reply:
I added currency as an argument. Do note that I changed the determination to default to EUR if nothing is set, and this default currency will now always be set on the transaction.

Note that tetermining whether the currency is set in the response is now done with an `isset` instead of `empty`. The tests didn't complain about it, but I'm not sure about possible edgecases missed.